diff --git a/.gitignore b/.gitignore
index ccea0c4..fa313dd 100644
--- a/.gitignore
+++ b/.gitignore
@@ -1,9 +1,5 @@
-.depend
-.settings
-.project
-.cproject
-/kern/compile/
-defs.mk
 build
-/user/testbin/randcall/calls.c
-*.swp
+.depend
+/defs.mk
+kern/compile/
+user/testbin/randcall/calls.c
diff --git a/kern/startup/main.c b/kern/startup/main.c
index be4c4b8..8098eb7 100644
--- a/kern/startup/main.c
+++ b/kern/startup/main.c
@@ -1,3 +1,4 @@
+//weihaoqu@buffalo.edu
 /*
  * Copyright (c) 2000, 2001, 2002, 2003, 2004, 2005, 2008, 2009
  *	The President and Fellows of Harvard College.
@@ -100,7 +101,11 @@ boot(void)
 	kprintf("%s", harvard_copyright);
 	kprintf("\n");
 
-	kprintf("Put-your-group-name-here's system version %s (%s #%d)\n", 
+<<<<<<< HEAD
+	kprintf("Weihaoqu's system version %s (%s #%d)\n", 
+=======
+	kprintf("Weihao's system version %s (%s #%d)\n", 
+>>>>>>> 5e48a1cead28c9d12fb4154f3c07f96b323c5cc3
 		GROUP_VERSION, buildconfig, buildversion);
 	kprintf("\n");
 
@@ -202,7 +207,7 @@ sys_reboot(int code)
  */
 void
 kmain(char *arguments)
-{
+{      
 	boot();
 
 	menu(arguments);
